tests/FakeItEasy.Specs/OrderedCallMatchingSpecs.cs

```diff
@@ -173,27 +222,40 @@ public interface IHaveOneGenericParameter
             "And a call on the Fake, passing argument 2"
                 .x(() => fake.Bar(2));
 
-            "And a call-ordering context"
-                .x(c => sequentialCallContext = A.SequentialCallContext());
+            "When I expect a call with argument 1 to have been made twice"
+                .x(() => exception = Record.Exception(() => lastAssertion = A.CallTo(() => fake.Bar(1)).MustHaveHappened(Repeated.Exactly.Twice)));

```

Comment:
I see your point. For some reason, I was thinking that storing the first exception was a good idea, but just letting the unexpected ones fly is probably better. Will make the change throughout.